providers/implementations/ciphers/cipher_sm4_ccm_hw.c

```diff
@@ -13,17 +13,40 @@
 
 #include "cipher_sm4_ccm.h"
 
```

Comment:
#include "crypto/sm4_platform.h" should be added here, otherwise below acceleration implementations can't be called.

Reply:
I'll check it

Reply:
done